spec/support/static_validation_helpers.rb

```diff
@@ -12,7 +13,8 @@
 module StaticValidationHelpers
   def errors
     target_schema = schema
-    validator = GraphQL::StaticValidation::Validator.new(schema: target_schema)
+    target_rules = defined?(rules) ? rules : GraphQL::StaticValidation::ALL_RULES
```

Comment:
While working on validation rule tests, I noticed we run validate against `ALL_RULES`.

I think it might be cleaner if each validation rule spec tested only against its own rules that way we could have stricter assertions.

@rmosolgo do you see this being a problem? If not, we could add `let(:rules) { .. }` to every spec and :fire: the `defined?(rules)` here.